en_US/configuration/configuration.md

```diff
@@ -654,6 +654,62 @@ NOTE: this config only takes effect when `rpc.port_discovery` is set to `manual`
 
 Set the number of RPC communication channels initiated by this node to each remote node. Set to 1 to ensure the order of messages. Keep the default value (half the number of CPU cores) to improve RPC throughput.
 
+### rpc.driver
+
+| Type    | Optional Value | Default         |
+| ------- | -------------- | --------------- |
+| enum    | `tcp`, `ssl`   | `tcp` |
+
+#### Description
+
+Transport-layer protocol used for communication between the brokers.
+
+
+### rpc.enable_ssl
```

Comment:
Name of this parameter is confusing, I don't remember why it's called like this in the schema, but this is how it looks in the released code :confused: